superset/migrations/shared/utils.py

```diff
@@ -43,11 +43,9 @@ def table_has_column(table: str, column: str) -> bool:
     :param column: A column name
     :returns: True iff the column exists in the table
     """
-    config = op.get_context().config
-    engine = engine_from_config(
-        config.get_section(config.config_ini_section), prefix="sqlalchemy."
-    )
-    insp = reflection.Inspector.from_engine(engine)
+
+    insp = inspect(op.get_context().bind)
```

Comment:
The `table_has_column` method was returning `False` for PostgreSQL within CI even though the columns existed. The proposed update is from the same [Stack Overflow](https://stackoverflow.com/questions/24082542/check-if-a-table-column-exists-in-the-database-using-sqlalchemy-and-alembic) post, however it's from 2022 (as opposed to 2018) and has more votes.